wordpress-mobile/WordPress-iOS
# Implement media upload from device in GB.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/471 

Related to the following PRs

 - in GB-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/485
 - in GB: https://github.com/WordPress/gutenberg/pull/13421

To test:
 - Open or create a post using GB mobile
 - Tap to add a new image to the post
  - Select upload on the block
  - Select image
  - Check if upload progress is reported
  - Check that the final image is updated.
 
Update release notes:
